examples/statistics/barchart_demo.py

```diff
@@ -28,9 +28,9 @@
 
 fig, ax = plt.subplots()
 rects1 = ax.bar(ind - width/2, men_means, width, yerr=men_std,
-                color='SkyBlue', label='Men')
+                label='Men')
```

Comment:
Maybe sexist, but one could argue that the colors had gender associations that help in reading the chart....

Reply:
The default colour cycle makes them different colours anyway: https://1164-74355158-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/gallery/statistics/barchart_demo.html#sphx-glr-gallery-statistics-barchart-demo-py

Reply:
Sure, but blue/light red have are often deemed masculine/feminine.  OTOH, I don't really feel strongly we need to perpetuate that.

Reply:
I agree, which is why I removed the explicit colours.